app/assets/javascripts/analytics/custom-dimensions.js

```diff
@@ -46,6 +46,7 @@
       'political-status': {dimension: 7},
       'analytics:organisations': {dimension: 9},
       'analytics:world-locations': {dimension: 10},
+      'publication-status': {dimension: 12, defaultValue: 'other'},
```

Comment:
Ok in that case the publication-status would need to be sent separately. I've renamed the meta tag to `withdrawn`.

Regarding the default value, the original request was to be able to filter out the `withdrawn` documents. I've been told by the analysis team that they prefer to have a default value so that they can easily filter not only "withdrawn" docs, but the opposite as well. This is because GA has a habit of not showing the data at all if the custom dimension value is not set.